src/reducers/fetch-reducer.ts

```diff
@@ -57,6 +58,15 @@ export function reducer(
             error: failAction.payload.error,
           });
           return;
+        case RESOURCE_DEL_SUCCESS:
+          const successDeleteAction = action as ResourceSuccessAction;
+          Object.assign(resource, {
```

Comment:
i think we might need to remove the whole resource from the store

Reply:
What about this instead `omit(draft, resourceUri);`?

Reply:
Changed